crispycuts/src/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import reportWebVitals from './reportWebVitals';
+
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <App />
+//   </React.StrictMode>,
+//   document.getElementById('root')
+// );
+
+// // If you want to start measuring performance in your app, pass a function
+// // to log results (for example: reportWebVitals(console.log))
+// // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+// reportWebVitals();
+
+const Body= ()=>{
+  return(
+    <div id="body">
```

Comment:
`id="body"` doesn't do anything. Since you didn't declare an id in `index.css`. The declaration in `index.css` for body is styling for the entire document which means every element in the dom tree will have that styling as a base.